### ISSUE TITLE
"combine" doesn't create non-existent destination directories while  "copy" does

### ISSUE THREAD

**Issue by hovsep** [JeffreyWay/laravel-mix#271]
The error after "npm run dev" is:

```
fs.js:558
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open 
```

**Comment by JeffreyWay** [comment]
I'll check with the plugin author to see if he can add an option to force-create the folder if it doesn't exist.

**Comment by JeffreyWay** [comment]
This is fixed in 8870ff16710c085e6fa3b1f98d1fba435efe0258
